rpcs3/Emu/RSX/Program/VertexProgramDecompiler.cpp

```diff
@@ -155,6 +155,7 @@ std::string VertexProgramDecompiler::GetSRC(const u32 n)
 	case 0: abs = d0.src0_abs; break;
 	case 1: abs = d0.src1_abs; break;
 	case 2: abs = d0.src2_abs; break;
+	default: fmt::throw_exception("n out of bounds (n=%d)", n);
```

Comment:
We'll have already fucked up if we get here with n>2. Makes more sense to just guard the entire function with an assert/ensure.